spigot/src/main/java/org/dynmap/bukkit/Helper.java

```diff
@@ -40,7 +40,7 @@ else if(Bukkit.getServer().getClass().getName().contains("GlowServer")) {
                 Log.info("Loading Glowstone support");
                 BukkitVersionHelper.helper = loadVersionHelper("org.dynmap.bukkit.helper.BukkitVersionHelperGlowstone");
             }
-            else if (v.contains("(MC: 1.21)")) {
+            else if (v.contains("(MC: 1.21)") || v.contains("(MC: 1.21.1)")) {
```

Comment:
This will break again when 1.21.2 is released. Just removing the ) will keep using the same helper:
`
            else if (v.contains("(MC: 1.21")) {
`